sentry_sdk/crons/decorator.py

```diff
@@ -47,15 +47,18 @@ def test(arg):
     ```
     """
 
-    def __init__(self, monitor_slug=None):
-        # type: (Optional[str]) -> None
+    def __init__(self, monitor_slug=None, monitor_config=None):
+        # type: (Optional[str], Optional[Dict[str, Any]]) -> None
```

Comment:
```suggestion
        # type: (Optional[str], Optional[dict[str, Any]]) -> None
```

Reply:
Even better would be to make `monitor_config` into a `TypedDict`; probably, this would be something for a separate PR, though.

Reply:
Will do this in a separate PR 👍🏻

Reply:
Issue here: #2930